pkg/strategy/support/strategy.go

```diff
@@ -439,7 +390,48 @@ func (s *Strategy) Run(ctx context.Context, orderExecutor bbgo.OrderExecutor, se
 	s.orderExecutor = orderExecutor
 
 	// StrategyController
-	s.status = types.StrategyStatusRunning
+	s.Status = types.StrategyStatusRunning
+
+	s.OnSuspend(func() error {
+		var err error = nil
+		// Cancel all order
+		for _, order := range s.orderStore.Orders() {
+			err = s.cancelOrder(order.OrderID, ctx, s.orderExecutor)
+		}
+		if err != nil {
+			errMsg := "Not all orders are cancelled! Please check again."
+			log.WithError(err).Errorf(errMsg)
+			s.Notify(errMsg)
+		} else {
+			s.Notify("All orders cancelled.")
```

Comment:
`All {symbol} orders are cancelled.`

Reply:
Because you might run multiple support strategies with different symbols.

Reply:
done